docs/docs-components/siteIndex.js

```diff
@@ -195,14 +195,18 @@ const siteIndex: $ReadOnlyArray<siteIndexType> = [
       },
       'Design tokens',
       'Elevation',
+      {
+        sectionName: 'Forms',
+        pages: ['Overview', 'Composition + behavior', 'Available components'],
```

Comment:
nit  

+ to and

Reply:
per content standards "+" is allowed. Was worried about wrapping, so I changed to "structure and behavior".

Reply:
thanks, yeah it was more for consistency. I don't think we do the `+` I only see `and`